README.md

```diff
@@ -1,110 +1,124 @@
 # The missing introduction to APCA
 
-## What is APCA?
+## _What is APCA in three sentences:_
+It is a supra threshold visual contrast prediction model that is specifically tuned to readability on self-illuminated displays. It follows human perception, including critical aspects such as spatial frequency, meaning font weight and size. The APCA guidelines also inform various use cases, such as body text, fluent text, spot reading text, and non-text items.
```

Comment:
Line length should be limited to 80 characters.

Reply:
Hard wrapping should NEVER be used.